src/macros.rs

```diff
@@ -0,0 +1,73 @@
+#[macro_export]
```

Comment:
Don't know why but the macros are not formatted by rustfmt. If you are using Vim, do an `=G` on this, please...